event_sync/mod_event_sync_component.lua

```diff
@@ -291,6 +301,7 @@ function occupant_joined(event)
     local payload = {
             ['event_name'] = 'muc-occupant-joined';
             ['occupant'] = occupant_data;
+            ['active_occupants'] = room_data:get_active_occupants();
```

Comment:
Same here. `active_occupant` could be misunderstood to mean array of occupant data, especially when there are fields like `occupant` and `all_occupants` that returns data. 

Perhaps `active_occupant_count`?